src/py-opentimelineio/opentimelineio/core/__init__.py

```diff
@@ -38,6 +38,32 @@
     item,
 )
 
+__all__ = [
+    'CannotComputeAvailableRangeError',
```

Comment:
I had to make a choice here. This exception is made available in both `opentimelineio.exceptions` and `opentimelineio.core`.

I decided to publicly export it here to remind me about this. So here I am, exposing the problem. I'd prefer to have the exception exported in the `exceptions` module and remove the import from here, but that could break users...

Reply:
I think we intended all of the exceptions to be in the `opentimelineio.exceptions` module.